journals/apps/journals/management/commands/publish_journals.py

```diff
@@ -85,8 +85,8 @@ def _update_wagtail_pages(self, journal, create=True, publish=True):
             # get base root page and add JournalIndexPage as child
             root_page = Page.get_root_nodes()[0]
             index_page = JournalIndexPage(
-                title="{} Index Page".format(site.site_name),
-                intro="{} introduction".format(site.site_name)
+                title="{} Index Page".format(site.site_name if site.site_name else journal.organization.name),
```

Comment:
NIT: Wouldn't we want this to be the other way around? In the case of a site of edX and multiple orgs (MITx, Harvardx), wouldn't we want it to show "MIT Index Page" instead of "edX Index Page" for both MIT and Harvard? It's almost guaranteed to be overwritten so it doesn't really matter, but just a thought.